Src/WitsmlExplorer.Api/Models/Reports/BaseReport.cs

```diff
@@ -9,5 +9,6 @@ public class BaseReport
         public IEnumerable<object> ReportItems { get; init; }
         public string WarningMessage { get; init; }
         public bool DownloadImmediately { get; init; } = false;
+        public string ReportHeader { get; init; } = null;
```

Comment:
You can remove = null as it should be null by default.

Reply:
fixed